mendix/widgets-resources
# [WC-97] Badge - Prepare release version 2

### Why?
Badge v1.x.x is a dojo widget that needs to be converted to a pluggable widget. It introduces breaking changes.

### What?
- Rename widget
- Add style design property (replaces brand style widget configuration property)
- Small PIW API usage improvements
- Fix preview prop usage
- Add structure preview
- Move badge styles to Atlas 3
- Use 8 grid system for badge
- Fix atlas release script output (add folder `atlas_ui_resources` in `themesource` folder)
- Refactor build script file to async/await + introduce nice messages
- Fix pluggable-widgets-tools e2e test script to always test on latest mx version when no `--mx-version` is supplied.

### How to test?
- Test on click and data update behaviour
- Verify structure preview (requires a custom build that fixes a bug)
- Verify design properties
- Verify preview (design properties not reflected in styles yet; page editor team will fix this)
